app/views/announcements/_announcement.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if current_announcement %>
+  <% if current_announcement.start_date <= Date.current && current_announcement.end_date >= Date.current %>
+    <div class="container content-wrapper">
+      <span><%== current_announcement.body %></span>
+      <% if current_announcement.link? %>
+        <%= link_to "Read more ->", current_announcement.link, class: "btn" %>
```

Comment:
You can use "primary-delete-button" class (present in components.scss). Also, maybe we can remove that arrow-head

Reply:
made changes accordingly !!